plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/inline/InlineChatEditorHint.kt

```diff
@@ -19,23 +19,26 @@ import javax.swing.JPanel
 
 class InlineChatEditorHint {
     private val hint = createHint()
+    private val hintXOffset = 20
```

Comment:
can you double check what happens when the UI is scaled really big?

Reply:
checked on both small window size and big monitor, it looks fine